src/System.Net.Security/src/System/Net/Security/SslConnectionInfo.OSX.cs

```diff
@@ -60,85 +91,797 @@ private struct TlsMapping
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA384,
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,
+                TlsMapping.EcDhEphem(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
             },
 
             {
                 TlsCipherSuite.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384,
-                TlsMapping.EcDhe(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+                TlsMapping.EcDhEphem(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDH_RSA_WITH_AES_128_CBC_SHA256,
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDH_RSA_WITH_AES_256_CBC_SHA384,
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+                TlsMapping.EcDhEphem(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
             },
 
             {
                 TlsCipherSuite.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
-                TlsMapping.EcDhe(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+                TlsMapping.EcDhEphem(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
             },
 
             {
+                TlsCipherSuite.TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256,
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDH_ECDSA_WITH_AES_256_GCM_SHA384,
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+            },
+            {
                 TlsCipherSuite.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
-                TlsMapping.EcDhe(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
+                TlsMapping.EcDhEphem(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
             },
 
             {
                 TlsCipherSuite.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
-                TlsMapping.EcDhe(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+                TlsMapping.EcDhEphem(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256,
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes128, HashAlgorithmType.Sha256)
+            },
+
+            {
+                TlsCipherSuite.TLS_ECDH_RSA_WITH_AES_256_GCM_SHA384,
+                TlsMapping.EcDhStatic(CipherAlgorithmType.Aes256, HashAlgorithmType.Sha384)
```

Comment:
No.  .NET doesn't allow the client CipherSuite order to be controlled, so the three that were already defined are pretty much all that will ever get hit on a Mac client (unless it's talking to convoluted scenarios).

Having spent a good 30 minutes verifying my 5 minutes of `%s/` commands in `vi`, I think the list is as good as it'll get.